pkg/ccl/logictestccl/tests/5node/5node_test: TestCCLLogic_partitioning_hash_sharded_index failed

pkg/ccl/logictestccl/tests/5node/5node_test.TestCCLLogic_partitioning_hash_sharded_index [failed](https://tanzanite.cluster.engflow.com/invocations/default/b6f34973-d6d6-42e0-88ff-c20eac9a0723?testReportRun=3&testReportShard=4&testReportAttempt=1#targets-Ly9wa2cvY2NsL2xvZ2ljdGVzdGNjbC90ZXN0cy81bm9kZTo1bm9kZV90ZXN0) on release-24.2.6-rc @ [9dab6cb9f71f21ef1e9c98640193035783ba9a4e](https://github.com/cockroachdb/cockroach/commits/9dab6cb9f71f21ef1e9c98640193035783ba9a4e):

Fatal error:

```
panic: test timed out after 1h16m37s
running tests:
	TestCCLLogic_partitioning_hash_sharded_index (1h16m37s)
	TestCCLLogic_partitioning_hash_sharded_index/test_presplit_with_partitioning (47m11s)

```
Stack: 

```
goroutine 2593110 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x265
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x45
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestCCLLogic_partitioning_hash_sharded_index
    test_log_scope.go:165: test logs captured to: outputs.zip/logTestCCLLogic_partitioning_hash_sharded_index1951954934
    test_log_scope.go:76: use -show-logs to present logs inline
[12:51:51] --- progress: /var/lib/engflow/worker/work/0/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/5node/5node_test_/5node_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/logictestccl/testdata/logic_test/partitioning_hash_sharded_index: 15 statements
[12:52:19] --- progress: /var/lib/engflow/worker/work/0/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/5node/5node_test_/5node_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/logictestccl/testdata/logic_test/partitioning_hash_sharded_index: 16 statements
[12:54:20] --- progress: /var/lib/engflow/worker/work/0/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/5node/5node_test_/5node_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/logictestccl/testdata/logic_test/partitioning_hash_sharded_index: 18 statements
[13:19:57] --- progress: /var/lib/engflow/worker/work/0/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/5node/5node_test_/5node_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/logictestccl/testdata/logic_test/partitioning_hash_sharded_index: 20 statements
[13:20:35] --- progress: /var/lib/engflow/worker/work/0/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/5node/5node_test_/5node_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/logictestccl/testdata/logic_test/partitioning_hash_sharded_index: 21 statements
[13:20:38] --- progress: /var/lib/engflow/worker/work/0/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/5node/5node_test_/5node_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/logictestccl/testdata/logic_test/partitioning_hash_sharded_index: 22 statements
[13:20:51] --- progress: /var/lib/engflow/worker/work/0/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/5node/5node_test_/5node_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/logictestccl/testdata/logic_test/partitioning_hash_sharded_index: 23 statements
[13:20:58] --- progress: /var/lib/engflow/worker/work/0/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/5node/5node_test_/5node_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/logictestccl/testdata/logic_test/partitioning_hash_sharded_index: 25 statements
=== RUN   TestCCLLogic_partitioning_hash_sharded_index/test_presplit_with_partitioning
```
</p>
</details>

Parameters:
 - <code>attempt=1</code>
 - <code>race=true</code>
 - <code>run=3</code>
 - <code>shard=4</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestCCLLogic_partitioning_hash_sharded_index.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>


Jira issue: CRDB-45294

--- a/pkg/sql/rowinfra/base.go
+++ b/pkg/sql/rowinfra/base.go
@@ -40,14 +40,14 @@ const ProductionKVBatchSize KeyLimit = 100000
 
 // defaultBatchBytesLimit is the maximum number of bytes a scan request can
 // return.
-var defaultBatchBytesLimit = BytesLimit(skip.ClampMetamorphicConstantUnderStress(
+var defaultBatchBytesLimit = BytesLimit(skip.ClampMetamorphicConstantUnderDuress(
 	metamorphic.ConstantWithTestRange(
 		"default-batch-bytes-limit",
 		defaultBatchBytesLimitProductionValue, /* defaultValue */
 		1,                                     /* min */
 		64<<10,                                /* max, 64KiB */
 	),
-	1<<10, /* min, 1KiB */
+	10<<10, /* min, 10KiB */
 ))
 
 const defaultBatchBytesLimitProductionValue = 10 << 20 /* 10MiB */

--- a/pkg/testutils/skip/constants.go
+++ b/pkg/testutils/skip/constants.go
@@ -5,11 +5,11 @@
 
 package skip
 
-// ClampMetamorphicConstantUnderStress ensures that the given integer constant
+// ClampMetamorphicConstantUnderDuress ensures that the given integer constant
 // with metamorphic testing range is at least the given minimum value, when the
-// process is running under stress.
-func ClampMetamorphicConstantUnderStress(val, min int) int {
-	if Stress() && val < min {
+// process is running under duress.
+func ClampMetamorphicConstantUnderDuress(val, min int) int {
+	if Duress() && val < min {
 		return min
 	}
 	return val

